### PR TITLE
Project Proposal

### PR DESCRIPTION
**Pull request by aundus**
Project proposal for Stevens group: Adam Undus, Anthony Ciccone, Damon Del Priore, John Dyer, Liam Nagel

### CODE REVIEW

**Comment by eshellman** [comment]
could you please edit the [/bookshelves.md file](/bookshelves.md)? pdf files don't lend themselves to version control or github display. Thanks

File: bookshelves.md [hunk 1]
```diff
@@ -8,39 +8,60 @@ The Free Ebook Foundation (FEF) has been helping Project Gutenberg (PG) renovate
...
 
 ## Goal
 
 Have prototype system running by April, 2021 for public review, move to production bin May.
 
+## Timeline
+
+| Dates               |                                                               Milestones                                                               |
+| ------------------- | :------------------------------------------------------------------------------------------------------------------------------------: |
+| 09/14/20 - 9/21/20  | Review Technical Specs, existing code/documentation, and implementation goals for deeper understanding of technical aspects of problem |
```
Comment (eshellman) [inline bookshelves.md:41]: What technical specs do you mean?

File: bookshelves.md [hunk 2]
```diff
@@ -8,39 +8,60 @@ The Free Ebook Foundation (FEF) has been helping Project Gutenberg (PG) renovate
...
+| 09/25/20            |   Deliver implementation plan and user stories for dev team to iteratively implement; allowing the dev team to reevaluate any cases.   |
+| 09/28/20 - 04/30/21 |    Main Development Period: Iteratively develop user stories several sprints. Demo new features at the end of sprints for feedback.    |
```
Comment (eshellman) [inline bookshelves.md:43]: While I have sketch out a possible "solution", there needs to be some feasibility evaluation. For the management app, there may be open-source off-the-shelf solutions.

File: bookshelves.md [hunk 3]
```diff
@@ -8,39 +8,60 @@ The Free Ebook Foundation (FEF) has been helping Project Gutenberg (PG) renovate
...
 ## Team
 
-Add names and links here.
+Adam Undus (aundus@stevens.edu)
+Liam Nagel (nagel@stevens.edu)
+John Dyer (jdyer@stevens.edu)
+Damon Del Priore (ddelprio@stevens.edu)
+Anthony Ciccone (aciccone@stevens.edu)
+
+## More about our team
```
Comment (eshellman) [inline bookshelves.md:57]: Do any of you have python experience?

File: bookshelves.md [hunk 4]
```diff
@@ -8,39 +8,60 @@ The Free Ebook Foundation (FEF) has been helping Project Gutenberg (PG) renovate
...
 Have prototype system running by April, 2021 for public review, move to production bin May.
 
+## Timeline
+
+| Dates               |                                                               Milestones                                                               |
+| ------------------- | :------------------------------------------------------------------------------------------------------------------------------------: |
+| 09/14/20 - 9/21/20  | Review Technical Specs, existing code/documentation, and implementation goals for deeper understanding of technical aspects of problem |
+| 09/25/20            |   Deliver implementation plan and user stories for dev team to iteratively implement; allowing the dev team to reevaluate any cases.   |
+| 09/28/20 - 04/30/21 |    Main Development Period: Iteratively develop user stories several sprints. Demo new features at the end of sprints for feedback.    |
+| 04/30/21            |                                                       Final demo of new features                                                       |
```
[the remark this answers is not in the document]
Comment (eshellman) [reply bookshelves.md:44]: I should also mention that deployability needs to be thought out early in the process.

File: bookshelves.md [hunk 5]
```diff
@@ -8,39 +8,60 @@ The Free Ebook Foundation (FEF) has been helping Project Gutenberg (PG) renovate
...
-    - Python/Flask or Python/CherryPy with Postgres backend
-    - Managers can make new bookshelves and categories
-    - Moderators can add/remove items from Builder Bookshelves
 
+  - HTML5 local storage + javascript
+  - Button on Item page
+  - Import/export lists for sharing
 
+- Bookshelf Management App
+  - Python/Flask or Python/CherryPy with Postgres backend
```
Comment (eshellman) [inline bookshelves.md:29]: Python/Django is also a reasonable choice. You can run Flask in CherryPy as well. Could do it with Node, but there's not much experience with JS apps on the PG IT team.

File: bookshelves.md (hunk 1 above)
Comment (eshellman) [reply bookshelves.md:41]: I would suggest adding a week to document requirements of PG deployment environment and evaluate technology options.

**Review by eshellman — CHANGES_REQUESTED** [review]
If you're looking for a 5th team member you might note that here, too.

Don't forget to think about the school year in proposing your schedule. (If you plan on taking other classes!)

File: bookshelves.md (hunk 3 above)
Comment (aundus) [reply bookshelves.md:57]: Yes, I believe all of us have some Python experience.

File: bookshelves.md (hunk 5 above)
Comment (aundus) [reply bookshelves.md:29]: Personally I am most comfortable with Node of those options, but that does not mean it is the best tool for the job. I think we would make the decision on this as we review the existing code and detail out some of the user stories. The decision would be finalized by the implementation plan in the updated timeline

File: bookshelves.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (aundus) [reply bookshelves.md:44]: Updated the timeline based on your comments to space out the planning and allows several more weeks for the delivery/QA portion.

**Comment by aundus** [comment]
@eshellman I updated the timeline based on your comments. Will add everyones github usernames shortly.

File: bookshelves.md [hunk 6]
```diff
@@ -8,39 +8,62 @@ The Free Ebook Foundation (FEF) has been helping Project Gutenberg (PG) renovate
...
+| 03/19/21            |                                                       Final demo of new features                                                       |
+| 03/19/21 - 04/30/21 |                              QA period to resolve issues and bugs. Final preparation for product release                               |
+
+_Dates/Milestones are tentative and subject to change._
+
 ## Team
 
-Add names and links here.
+Adam Undus (aundus@stevens.edu)
+Liam Nagel (nagel@stevens.edu)
```
Comment (eshellman) [inline bookshelves.md:54]: Your invitation to nagel@stevens.edu has bounced.

Comment (aundus) [reply bookshelves.md:54]: Made a typo in the email fix here: https://github.com/EbookFoundation/capstone-projects/pull/10
I resent an invite to him.

**Pull request merged**